chapters/data-analysis.tex

```diff
@@ -1,182 +1,236 @@
+a data set called \texttt{section-1-clean}.
 
 The division of a project in stages also helps the review workflow inside your team.
 The code, data and outputs of each of these stages should go through at least one round of code review.
 During the code review process, team members should read and run each other's codes.
 Doing this at the end of each stage helps prevent the amount of work to be reviewed to become too overwhelming.
 Code review is a common quality assurance practice among data scientists.
-It helps to keep the level of the outputs high, and is also a great way to learn and improve your code.
+It helps to keep the quality of the outputs high, and is also a great way to learn and improve your own code.
+
+\subsection{Writing master scripts}
 
-\subsection{Master scripts}
 Master scripts allow users to execute all the project code from a single file.
-They briefly describes what each code, 
-and maps the files they require and create. 
-They also connects code and folder structure through globals or objects. 
-In short, a master script is a human-readable map to the tasks, 
-files and folder structure that comprise a project.  
-Having a master script eliminates the need for complex instructions to replicate results. 
-Reading the master do-file should be enough for anyone unfamiliar with the project
+They briefly describe what each code does,
+and map the files they require and create.
+They also connect code and folder structure through macros or objects.
+In short, a master script is a human-readable map of the tasks,
+files, and folder structure that comprise a project.
+Having a master script eliminates the need for complex instructions to replicate results.
+Reading it should be enough for anyone unfamiliar with the project
 to understand what are the main tasks, which scripts execute them,
-and where different files can be found in the project folder. 
+and where different files can be found in the project folder.
 That is, it should contain all the information needed to interact with a project's data work.
 
-\subsection{Version control}
-Finally, everything that can be version-controlled should be. 
+\subsection{Implementing version control}
+
+Establishing a version control system is an incredibly useful
+and important step for documentation, collaboration and conflict-solving.
 Version control allows you to effectively track code edits,
-including the addition and deletion of files. 
-This way you can delete code you no longer need, 
+including the addition and deletion of files.
+This way you can delete code you no longer need,
 and still recover it easily if you ever need to get back previous work.
+Everything that can be version-controlled should be.
 Both analysis results and data sets will change with the code.
-You should have each of them stored with the code that created it.
-If you are writing code in Git/GitHub,
+Whenever possible, you should track have each of them with the code that created it.
```

Comment:
```suggestion
Whenever possible, you should track each of them with the code that created it.
```

Reply:
Extra word that should be removed, but even after removing the word, I do not know what we want to say in this sentence.

Reply:
What I wanted to say it that ideally, when you make changes to a particular script, you then run all the other scripts linked to it from the master, and the commit includes not only changes to the code, but also to the outputs and datasets it may create. But I see your point that this is not super clear. Does my explanation make sense to you?

Reply:
done in 28cc3f469ffc3ad1c2e7d38e9f08c88185970dc7